### ISSUE TITLE
NameError raised by dendropy.calculate.treecompare.find_missing_bipartitions

### ISSUE THREAD

**Issue by daveuu** [jeetsukumaran/DendroPy#26]
Hi

In version 4.0.3 a NameError is raised by find_missing_bipartitions(), unless I'm doing something wrong. Can be reproduced with:
```
tree1 = dendropy.Tree.get_from_string('(A,(B,C));', 'newick')
tree2 = dendropy.Tree.get_from_string('((A,B),C);', 'newick', taxon_namespace = tree1.taxon_namespace)
dendropy.calculate.treecompare.find_missing_bipartitions(tree1, tree2)
```
which gives me
```
DendroPy-4.0.3/dendropy/calculate/treecompare.pyc in find_missing_bipartitions(reference_tree, comparison_tree, is_bipartitions_updated)
    340     if not is_bipartitions_updated:
    341         reference_tree.encode_bipartitions()
--> 342         comparision_tree.encode_bipartitions()
    343     else:
    344         if reference_tree.bipartition_encoding is None:

NameError: global name 'comparision_tree' is not defined
```

thanks for all the hard work! Version 4 looks good.

**Comment by daveuu** [comment]
Actually it's just a typo:
```
comparision_tree.encode_bipartitions()
       ^^^
```
should be
```
comparison_tree.encode_bipartitions()
```

**Comment by jeetsukumaran** [comment]
Thanks. Fixed in 5009324.

On 8/21/15 1:35 PM, daveuu wrote:
> Actually it's just a typo:
>
> |comparision_tree.encode_bipartitions() ^^^ |
>
> should be
>
> |comparison_tree.encode_bipartitions() |
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeetsukumaran/DendroPy/issues/26#issuecomment-133506118>.
>

-- 



--------------------------------------
Jeet Sukumaran
--------------------------------------
jeetsukumaran@gmail.com
--------------------------------------
Blog/Personal Pages:
    http://jeetworks.org/
GitHub Repositories:
    http://github.com/jeetsukumaran
Photographs (as stream):
    http://www.flickr.com/photos/jeetsukumaran/
Photographs (by galleries):
    http://www.flickr.com/photos/jeetsukumaran/sets/
--------------------------------------
